src/components/Chatbot/Chatbot.tsx

```diff
@@ -23,46 +19,115 @@ const NewConversationButton = styled(PrimaryButton)(() => ({
 }));
 
 export const Chatbot = () => {
-  const [conversation, setConversation] = useState<Message[]>([]);
+  const [conversation, setConversation] = useState<MessageBubbleProps[]>([]);
   const [error, setError] = useState(false);
   const [loading, setLoading] = useState(false);
+  const [conversationCreated, setConversationCreated] = useState<string | null>(
+    null
+  );
+
+  const loadConversationFromBackend = async () => {
+    try {
+      await loadConversation().then((response) => {
+        if (response.ok) {
+          setConversation(response.conversation.messages);
+          setConversationCreated(response.conversation.created_at.slice(0, 10));
+        } else {
+          setConversation([]);
+        }
+      });
+    } catch (error) {
```

Comment:
`loadConversation` function can never fail. It returns `ok: false`. Therefore even if on the preview URL (https://polite-meadow-040edd103-15.westeurope.3.azurestaticapps.net/) the loading of the conversation from the backend fails, no error is shown.
Can you please fix this and also add test for this? (mocking failure when loading previous conversation)

Reply:
I can't find the test.